### DISCUSSION TITLE
Emoji in URL breaks port forwarding service

### DISCUSSION THREAD

**Discussion by jdherg** [community/community#3748] [Codespaces]
Hi! Apologies if there's a better place for filing Codespaces bug reports -- I had trouble finding one. Feel free to reroute me!

A service I'm working on makes use of emoji in the URL, and when trying to use emoji (or some other less-used Unicode characters) I get a 500. This particular bug isn't with my service (which works with the given emoji locally and in production) so I assume it's something about the Codespaces port forwarding service.

Thanks!

**Comment by tanmayeekamath** [comment]
Hey @jdherg thanks for reporting the issue. I have filed a bug internally and will keep you updated on the progress. Thanks for your patience.
